lib/tmdb/client.rb

```diff
@@ -201,20 +233,50 @@ def get_parsed_tv_episode_data(series_id:, season_number:, episode_number:)
         JSON.parse(open(url).read, symbolize_names: true)
       end
 
-      def get_parsed_tv_search_results(query)
-        url = "#{BASE_URL}/search/tv?api_key=#{API_KEY}&query=#{query}"
+      def get_parsed_tv_search_results(series_name)
+        searchable_name = I18n.transliterate(series_name)
+        url = "#{BASE_URL}/search/tv?api_key=#{API_KEY}&query=#{searchable_name}"
         JSON.parse(open(url).read, symbolize_names: true)&.dig(:results)
       end
 
       def get_parsed_multi_search_results(query)
-        url = "#{BASE_URL}/search/multi?api_key=#{API_KEY}&query=#{query}"
+        searchable_query = I18n.transliterate(query)
+        url = "#{BASE_URL}/search/multi?api_key=#{API_KEY}&query=#{searchable_query}"
         JSON.parse(open(url).read, symbolize_names: true)&.dig(:results)
       end
 
-      def get_parsed_movie_search_results(query)
-        url = "#{BASE_URL}/search/movie?api_key=#{API_KEY}&query=#{query}"
+      def url_for_person_search(person_name)
+        searchable_name = I18n.transliterate(person_name)
+        "#{BASE_URL}/search/person?api_key=#{API_KEY}&query=#{searchable_name}"
+      end
+
+      def get_parsed_movie_search_results(movie_title)
+        searchable_title = I18n.transliterate(movie_title)
+        url = "#{BASE_URL}/search/movie?api_key=#{API_KEY}&query=#{searchable_title}"
         JSON.parse(open(url).read, symbolize_names: true)&.dig(:results)
       end
```

Comment:
When I switched to the `person_url` & `person_data` approach from the `get_parsed_person_search_results` approach, I needed to move this `I18n.transliterate` method into all of these other places. It's not super relevant to the rest of the work in this PR, but I wanted to call it out because it you may otherwise wonder what in the heck is going on here.